README.md

```diff
@@ -14,6 +14,7 @@ $('#foo').sticky({
 	topSpacing: 80, // Space between element and top of the viewport
 	zIndex: 100, // z-index
 	stopper: "#bar" // Id, class, or number value
+	stickyClass: 'is-sticky' // Class applied to element when it's stuck
```

Comment:
I like the idea. Let's make this feature optional to avoid unnecessary DOM touches. This can be the default className which the user can override. Using this feature can be a boolean, something like `useStickyClass: true`.